CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
+
+## How can I contribute?
+
+### Submitting an issue
+
+If you have discovered a bug, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
+
+### Starting a discussion
+
+If you want to share ideas, questions, or concerns with the USWDS core team and community, [open a discussion on GitHub](https://docs.github.com/en/discussions/quickstart#creating-a-new-discussion).
+
+### Submitting a pull request
+
+At this time, we are not accepting code contributions in the community. We expect to open up community pull request submissions in early 2025, so check back later for instructions on how to contribute pull requests to USWDS web components.
```

Comment:
**note:** I took some liberties in assuming a possible PR acceptance timeline here. The goal was to add some clarity and set some expectations (express that eventually we will want contributions - they won't be blocked forever). Please comment if you think this is unnecessary or unadvised.

Reply:
I think it sets expectations, which is good to me.